hub/constraints/generalities/context.lisp

```diff
@@ -61,10 +61,13 @@
                     (hub-stamp-constancy CN)
                     (hub-stamp-constancy CN_NEW)))
 
+(defconstraint    generalities---context-numbers---TX_EXEC-phase---CN-vanishes-outside-of-execution-phase ()
+                  (if-zero TX_EXEC
+                           (vanishes! CN)))
```

Comment:
This constraint is important for the validity of the Execution Environment Consistency argument to work as intended.